### ISSUE TITLE
NAACL-2019-CommonsenseQA: A Question Answering Challenge Targeting Commonsense Knowledge

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#104]
**一句话总结：**

基于先验知识的常识QA数据集。BERT-large也才56% accuracy，离人类的89%很远。

(NAACL-2019  Best Resource Paper)

**资源：**

- [pdf](<https://arxiv.org/abs/1811.00937>)
- [code](<https://github.com/jonathanherzig/commonsenseqa>)
- [dataset](<https://www.tau-nlp.org/commonsenseqa>)
- [paper-with-code](<https://paperswithcode.com/paper/commonsenseqa-a-question-answering-challenge>)

**关键字：**

- dataset: COMMONSENSEQA, a challenging new dataset for commonsense question answering.
- To investigate question answering with prior knowledge, we present COMMONSENSEQA
- To capture common sense beyond associations, we extract from CONCEPTNET (Speer et al., 2017) multiple target concepts that have the same semantic relation to a single source concept.
- BERT-large (Devlin et al., 2018) and obtains 56% accuracy, well below human performance, which is 89%

**笔记：**

用户在是有前验知识的前提下进行回答问题的。

When humans answer questions, they capitalize on their common sense and background knowledge about spatial relations, causes and effects, scientific facts and social conventions. For instance, given the question “Where was Simon when he heard the lawn mower?”, one can infer that the lawn mower is close to Simon, and that it is probably outdoors and situated at street level. This type of knowledge seems trivial for humans, but is still out of the reach of current natural language understanding (NLU) systems

人类在听到问题的时候，总会根据问题进行一系列的推理和联想。这对于人类很普通，但是对于natural language understanding (NLU) systems来说就不可能了。

大部分QA方面的工作都是回答factoid questions, where the answer can be found in a given context with little need for commonsense knowledge。这种情景下不怎么需要commonsense knowledge。

最近，基于常识推理的数据变多了。efforts have been invested in developing large-scale datasets for commonsense reasoning. 

COMMONSENSEQA也是其一。A new dataset focusing on commonsense question answering, based on knowledge encoded in CONCEPTNET (Speer et al., 2017).

We propose a method for generating commonsense questions at scale by asking crowd workers to author questions that describe the relation between concepts from CONCEPTNET (Figure 1). 

![image-20190412102804886](https://ws4.sinaimg.cn/large/006tNc79gy1g1zlev65enj30gz0kq0xa.jpg)

Because questions are generated freely by workers, they often require background knowledge that is trivial to humans but is seldom explicitly reported on the web due to reporting bias (Gordon and Van Durme, 2013). Thus, questions in COMMONSENSEQA have a different nature compared to prior QA benchmarks, where questions are authored given an input text.

We collected 12,247 commonsense questions. We find that fine-tuning BERT-LARGE
(Devlin et al., 2018) on COMMONSENSEQA obtains the best performance, reaching an accuracy
of 55.9%. This is substantially lower than human
performance, which is 88.9%.

To summarize, our contributions are:

1. A new QA dataset centered around common
   sense, containing 12,247 examples.
2. A new method for generating commonsense
   questions at scale from CONCEPTNET.
3. An empirical evaluation of state-of-the-art
   NLU models on COMMONSENSEQA, showing that humans substantially outperform current models.

**模型图：**

![image-20190412104235927](https://ws1.sinaimg.cn/large/006tNc79gy1g1zltjmljfj30ir0ie42g.jpg)

**结果**：

![image-20190412104318960](https://ws2.sinaimg.cn/large/006tNc79gy1g1zlueli6qj30yt09ydkj.jpg)

![image-20190412104417750](https://ws2.sinaimg.cn/large/006tNc79gy1g1zlvcefrej30yl0i0wsx.jpg)

![image-20190412104455906](https://ws2.sinaimg.cn/large/006tNc79gy1g1zlw2rd2dj30l00eatbx.jpg)
